SPECS/nlohmann-json/nlohmann-json.spec

```diff
@@ -0,0 +1,55 @@
+Summary:        Modern C++11 JSON library
+Name:           nlohmann-json
+Version:        3.6.1
+Release:        2%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment
+URL:            https://github.com/nlohmann/json
+#Source0:       https://github.com/nlohmann/json/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+%global debug_package %{nil}
```

Comment:
I think we can remove that. We'd like to have debug symbols for as many packages as we can.